geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/DeployToMultiGroupDUnitTest.java

```diff
@@ -64,10 +65,10 @@ public static void beforeClass() throws Exception {
     server1 = cluster.startServerVM(1, "group1", locator.getPort());
     server2 = cluster.startServerVM(2, "group2", locator.getPort());
 
-    client =
-        ClusterManagementServiceBuilder.buildWithHostAddress()
-            .setHostAddress("localhost", locator.getHttpPort())
-            .build();
+    client = new ClusterManagementServiceBuilder().setConnectionConfig(
+        new BasicClusterManagementServiceConnectionConfig("localhost", locator.getHttpPort()))
```

Comment:
"BasicClusterManagementServiceConnectionConfig" seems like an awful long name? Any thing shorter and sweeter? Preferably not something "Basic" :-).

Maybe just ConnectionConfig and CacheConnectionConfig

Reply:
It does seem pretty cumbersome. I changed the names to be shorter.

Reply:
Maybe instead of ConnectionConfigImpl which implies it's an internal implementation, change it to "BaseConnectionConfig"?

Reply:
Done